src/i2p.cpp

```diff
@@ -355,29 +366,48 @@ void Session::CreateIfNotCreatedAlready()
         return;
     }
 
-    Log("Creating SAM session with %s", m_control_host.ToString());
+    const auto session_type = m_transient ? "transient" : "persistent";
+
+    Log("Creating %s SAM session with %s", session_type, m_control_host.ToString());
```

Comment:
It might be handy to log the session id in this message to make it easier to match the session messages in a busy i2p log.

Reply:
Nice, done!